<?xml version="1.0"?>
<launch>
	<node name="static_path" type="static_path_node" pkg="path_planner" output="screen" respawn="true">
        <param name="use_map_topic" value="false" />
        <param name="map_service" value="/dynamic_map" />
        <param name="map_topic" value="/global_cost_map" />

        <param name="resolution" value="0.1" />

		<rosparam param="segments">[[4.0], [3.141592, 2.0], [2.0], [-3.141592, 2.0], [4.0]]</rosparam>		

	</node>
</launch>
